sigal/themes/galleria/templates/index.html

```diff
@@ -267,6 +262,20 @@ <h2>
     {% endif %}
 
     Galleria.ready(function() {
+	/*
+	load thumbnails after gallery setup, and download has started
+	for the main images, but before the loading of map tiles starts
+	in the background. This ensures visible elements are prioritised
+	over non-visible elements
+	
+	The choice to lazy load thumbnails in batches of 10 was not made
+	scientifically. Other numbers may yield better performance.
+	*/
+	this.lazyLoadChunks( 10 );
+    {% if settings.show_map and album.show_map %}
+    $(".gallery-map-toggle").click(galleryMapToggle);
+    {% endif %}
```

Comment:
Why is this necessary or useful ? 
Also  for me the map does not show up on your branch.

Reply:
Looking more in detail I see that it is already done a few lines above. So it should probably not been done twice ?
Actually I wonder if all the map loading could be done later, when clicking on the map button, to avoid loading tiles for nothing. But it's not related to your PR, I can try to address this later.

Reply:
The duplication was unintentional. Thanks for catching it. Instead of continuing to use this new `show_map` conditional, I’ve moved this simulated click into an existing conditional block that achieves the same outcome.

Reply:
It would be great to avoid unnecessary loading of the maps. There’s quite some data being downloaded in the background. I think the map function will be used infrequently enough that we shouldn’t unconditionally load it in the background.

Reply:
Agreed!